crates/flipperzero/src/gui/mod.rs

```diff
@@ -1,3 +1,230 @@
 //! GUI service.
 
 pub mod canvas;
+pub mod icon;
+pub mod icon_animation;
+pub mod view;
+pub mod view_port;
+pub mod xbm;
```

Comment:
These new modules should be mentioned in the changelog under the `Added` section.